dev/DynamicDependency/API/MsixDynamicDependency.h

```diff
@@ -191,7 +191,7 @@ STDAPI MddGetIdForPackageDependencyContext(
 
 #if defined(WINDOWSAPPSDK_MICROSOFT_WINDOWS_APPLICATIONMODEL_DYNAMICDEPENDENCY_FEATURE_GENERATIONID_ENABLED) && (WINDOWSAPPSDK_MICROSOFT_WINDOWS_APPLICATIONMODEL_DYNAMICDEPENDENCY_FEATURE_GENERATIONID_ENABLED == 1)
 /// Return the package graph's current generation id.
-UINT32 MddGetGenerationId() noexcept;
+STDAPI_(UINT32) MddGetGenerationId() noexcept;
```

Comment:
@MikeHillberg was tracking important changes in 1.0, this is one of them since it meaningfully changes the signature.